tests/foreman/cli/test_computeresource_rhev.py

```diff
@@ -439,46 +422,75 @@ def test_positive_provision_rhev_with_host_group(rhev, provisioning, tear_down):
 
     :CaseAutomation: Automated
     """
-    name = gen_string('alpha')
+    cli = module_provisioning_sat.sat.cli
+    cr_name = gen_string('alpha')
+    org_name = module_org_with_manifest.name
+    loc_name = module_location.name
     rhv_cr = ComputeResource.create(
         {
-            'name': name,
+            'name': cr_name,
             'provider': 'Ovirt',
             'user': rhev.username,
             'password': rhev.password,
             'datacenter': rhev.datacenter,
             'url': rhev.hostname,
             'ovirt-quota': rhev.quota,
-            'organizations': provisioning.org_name,
-            'locations': provisioning.loc_name,
+            'organizations': org_name,
+            'locations': loc_name,
+        }
+    )
+    assert rhv_cr['name'] == cr_name
+    domain_name = module_provisioning_sat.domain.name
+    subnet_name = module_provisioning_sat.subnet.name
+    hostgroup = cli.HostGroup.create(
+        {
+            'name': gen_string('alpha'),
+            'organization': org_name,
+            'location': loc_name,
+            'architecture': default_architecture.name,
+            'domain': domain_name,
+            'subnet': subnet_name,
+            'content-source-id': module_provisioning_sat.domain.dns.id,
+            'content-view-id': module_org_with_manifest.default_content_view.id,
+            'kickstart-repository-id': module_provisioning_rhel_content.ksrepo.id,
+            'lifecycle-environment-id': module_org_with_manifest.library.id,
+            'operatingsystem': module_provisioning_rhel_content.os.title,
+            'pxe-loader': "PXELinux BIOS",
+            'partition-table': default_partitiontable.name,
+            'compute-resource-id': rhv_cr.get('id'),
         }
     )
-    assert rhv_cr['name'] == name
     host_name = gen_string('alpha').lower()
-    host = make_host(
+    host = cli.Host.create(
         {
             'name': f'{host_name}',
-            'root-password': gen_string('alpha'),
-            'organization': provisioning.org_name,
-            'location': provisioning.loc_name,
+            'organization': org_name,
+            'location': loc_name,
+            'root-password': rhev.root_password,
```

Comment:
Can you use `settings.provisioning.host_root_password` here?

Reply:
I can and I should - however well I'm trying to look, I don't even see `rhev.root_password` defined anywhere in repo, that's probably something I put in the config manually and forgot to remove it.